databricks/koalas/tests/test_series.py

```diff
@@ -909,3 +909,33 @@ def test_keys(self):
         pser = kser.to_pandas()
 
         self.assert_eq(kser.keys(), pser.keys())
+
+    def test_pct_change(self):
+        kser = ks.Series([90, 91, 85], index=[2, 4, 1])
+        pser = kser.to_pandas()
+
+        self.assert_eq(repr(kser.pct_change(periods=-1)),
+                       repr(pser.pct_change(periods=-1)))
+        self.assert_eq(repr(kser.pct_change(periods=-100000000)),
+                       repr(pser.pct_change(periods=-100000000)))
+        self.assert_eq(repr(kser.pct_change(periods=100000000)),
+                       repr(pser.pct_change(periods=100000000)))
```

Comment:
Why are these using `repr()`?

Reply:
Because they raise `AssertionError` even they looks like same result.

![스크린샷 2019-12-03 오전 10 10 44](https://user-images.githubusercontent.com/44108233/70007946-7f772500-15b5-11ea-9292-4da397053774.png)

but i just changed them to use `almost=True` rather than `repr`